qdax/tasks/jumanji_envs.py

```diff
@@ -0,0 +1,189 @@
+from functools import partial
+from typing import Any, Callable, Tuple
+
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+import jumanji
+
+from qdax.core.neuroevolution.buffers.buffer import QDTransition, Transition
+from qdax.types import (
+    Descriptor,
+    ExtraScores,
+    Fitness,
+    Genotype,
+    JumanjiState,
+    JumanjiTimeStep,
+    Observation,
+    Params,
+    RNGKey,
+)
+
+
+def make_policy_network_play_step_fn(
```

Comment:
Can this be renamed `make_policy_network_play_step_fn_jumanji` (or something equivalent) to avoid any ambiguity with the Brax function?